admin/tools/docker/4_build-configured-images.sh

```diff
@@ -72,10 +69,9 @@ DOCKERFILE="$DOCKERDIR/Dockerfile"
 cp "$DOCKERDIR/Dockerfile.tpl" "$DOCKERFILE"
 sed -i 's%{{NODE_TYPE_OPT}}%-m%g' "$DOCKERFILE"
 sed -i "s%{{DOCKER_IMAGE_OPT}}%$DOCKER_IMAGE%g" "$DOCKERFILE"
-sed -i "s%{{MASTER_FQDN_OPT}}%${MASTER}%g" "$DOCKERFILE"
 sed -i 's%{{COMMENT_ON_WORKER_OPT}}%%g' "$DOCKERFILE"
 
-TAG="${DOCKER_IMAGE}_master_${MASTER}"
+TAG="${DOCKER_IMAGE}_master"
```

Comment:
`$(MASTER)` used to be appended in the image name for both czar and worker. Is it not anymore? If so, is it encompassed in `$(DOCKER_IMAGE)` now?